### ISSUE TITLE
Use full text label "Refresh" on refresh button instead of just icon

### ISSUE THREAD

**Issue by kwwii** [rancher/dashboard#8645]
For improved accessibility, always use a text label in buttons

**Comment by gaktive** [comment]
@aalves08 which components or modals (or pages)? During triage, we want to understand if how comprehensive this issue is so we can figure out if we can still do this for Q4.

**Comment by aalves08** [comment]
@gaktive It's just adding the text "refresh" to the manual refresh button and probably remove the tooltip (check screenshot - button showing tooltip). @kwwii do we remove the icon or do we keep it?

It should be a 0.5 size task, imo.

![Screenshot 2023-08-03 at 14 49 52](https://github.com/rancher/dashboard/assets/97888974/81cc43d8-a309-47e2-8e37-5cf3a6aa3161)

**Comment by gaktive** [comment]
@kwwii did any of our other button tickets cover this already?

**Comment by kwwii** [comment]
@gaktive No, this is a singular issue left over from some earlier work. It was identified during a demo iirc.

**Comment by kwwii** [comment]
@aalves08 The icon should remain (similar to the other table action buttons)
